src/Button/index.d.ts

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { ButtonProps } from 'react-bootstrap/Button';
+import { ButtonGroupProps, ButtonToolbarProps } from "react-bootstrap";
+import { BsPrefixRefForwardingComponent } from "react-bootstrap/helpers";
+
+export interface BaseButtonProps extends ButtonProps {
```

Comment:
I suggest using the "Base" prefix for types or interfaces imported from React Bootstrap. Because in this case, ButtonProps is the main interface, and our props interface extends the base interface of this React Bootstrap component.